common/app/model/FaciaDisplayElement.scala

```diff
@@ -45,3 +47,10 @@ case class CrosswordSvg(id: String) extends FaciaDisplayElement {
 
   def imageUrl = s"${Configuration.ajax.url}/$id.svg"
 }
+
+object InlineSlideshow {
+  def fromTrail(trail: Trail): InlineSlideshow =
+    InlineSlideshow(trail.trailSlideshow(5, 3))
```

Comment:
Discussed quickly with Jon, but maybe we could pass named aspect ratio vars instead of 5 and 3?